commands/import.go

```diff
@@ -40,27 +41,29 @@ func init() {
 type (
 	//Importer ...
 	Importer struct {
-		res            *resources.Resources
-		configFile     string
-		args           cli.Args
-		importFiles    []string
-		targetDatabase string
-		rolling        bool
-		totalChunks    int
-		currentChunk   int
-		threads        int
+		res             *resources.Resources
+		configFile      string
+		args            cli.Args
+		importFiles     []string
+		targetDatabase  string
+		deleteOldData   bool
+		userRolling     bool
+		userTotalChunks int
+		userCurrChunk   int
```

Comment:
deleteOldData is a new bool which contains the deleteFlag result.

rolling, totalChunks, and currentChunk were prefixed with the tag `user` to help distinguish their values between the values loaded from the metadb later on.